### PR TITLE
Add the Works and About me Sections

### PR DESCRIPTION
**Pull request by Benmuiruri**
## Task
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

## Changes

- Added "Works" section
- Added  "About myself" section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,6 +67,268 @@ <h1>
           </div>
         </div>
       </section>
+      <section class="works">
+        <div class="works-wrapper">
+          <div class="work-tile">
```
Comment (thecodechaser) [inline index.html:72]: - [x] Good job working on the design but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. You are missing the "My recent works" heading from the design, Please place the heading before projects and make sure it matches the design.

See this image for reference

Screenshot (66)

File: index.html [hunk 2]
```diff
@@ -67,6 +67,268 @@ <h1>
...
+                <button class="btn view-work"><span>See project</span></button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-wrapper">
+          <div class="about-content">
+            <h2>About me</h2>
```
Comment (thecodechaser) [inline index.html:257]: - [ ] You did a great job working on the about section, But your about section is missing the background vector images from the design, Please place background vector images in your about section and make sure it matches the figma design.

Your design

Screenshot (69)

Figma design

Screenshot (67)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Benmuiruri 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a title and summary. :heavy_check_mark:
- Good commit messages. :heavy_check_mark:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benmuiruri** [comment]
Hi @thecodechaser thanks for the review, I'll make the changes shortly.

File: index.html (hunk 1 above)
Comment (Benmuiruri) [reply index.html:72]: Thanks, I had missed this header.

**Comment by Benmuiruri** [comment]
HI @thecodechaser I have added the background image and works section header. Thanks

**Review by prolajumokeoni — APPROVED** [review]
Hi @Benmuiruri , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

_Check inline comments_
-
giphy



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benmuiruri** [comment]
Hi @prolajumokeoni  and @thecodechaser  thank you for the review. I will Merge and make the optional changes suggested by @prolajumokeoni . 

Cheers.

**Comment by prolajumokeoni** [comment]
> Hi @prolajumokeoni and @thecodechaser thank you for the review. I will Merge and make the optional changes suggested by @prolajumokeoni .
> 
> Cheers.

OK great, kindly tag me if you need more clarifications about the changes requested 👍

**Pull request merged**